kpops/component_handlers/helm_wrapper/utils.py

```diff
@@ -1,20 +1,26 @@
+import hashlib
 import logging
 
 log = logging.getLogger("HelmUtils")
 
-
+ENCODING = "utf-8"
 RELEASE_NAME_MAX_LEN = 52
 
 
-def trim_release_name(name: str, suffix: str = "") -> str:
-    """Trim Helm release name while preserving suffix.
+def create_helm_release_name(name: str) -> str:
```

Comment:
One thing to note here:
The release name of clean jobs will no longer include the suffix `-clean`. The release name of the app and the clean job will differ (they will have different hash values). This would be okay because the deployment name would be overridden (fullnameOverride) and contain the `-clean` suffix.
See tests for an example.

Reply:
@philipp94831 interested to hear your opinion on this, does that suffice or do you have to be able to distinguish them by the Helm release names?

Reply:
I would like the helm release name to always have -clean as a suffix. I think if there is a slight difference in the remaining name, that is no problem. Can we make clean release names like this? my-long-name-with-a-hash-ab435-clean?